readme/plugins.md

```diff
@@ -1,7 +1,27 @@
 # Plugins
 
-Joplin supports plugins, which can be used to add new features or modify the application behaviour.
+The **desktop app** has the ability to extend beyond its standard functionality by the way of plugins. These plugins adhere to the Joplin plugin API and can be installed & configured within the application via the `Plugins` page of the Settings menu (*Windows/Linux*: Tools > Options > Plugins; *macOS*: Joplin > Preferences). This menu allows the manual installation of the plugin using the single 'Joplin Plugin Archive' (*.jpl) file. Once the application is reloaded the plugins will appear within the plugins menu where they can be toggled on/off or removed entirely.
+
+## Plugin Repository
+
+Plugins are currently maintained by the community in the [Joplin Discourse 'plugins' category](https://discourse.joplinapp.org/c/plugins/18). These plugins are not officially tested or vetted and care should be taken to understand any potential risks that come from installation.
```

Comment:
Again please remove the warning. When it comes to security we need to be precise - what are the risks, what precautions do we take, etc. Vague statements like this aren't a good idea.